### ISSUE TITLE
no-multi-spaces conflicts with key-spacing in align mode

### ISSUE THREAD

**Issue by btmills** [eslint/eslint#1755]
```js
// test.js
/* eslint no-multi-spaces: 2, key-spacing: [2, { align: "value" }] */
var obj = {
	first:  "first",
	second: "second"
};
```

```sh
$ eslint --reset --no-eslintrc test.js

test.js
  4:9  error  Multiple spaces found before '"first"'  no-multi-spaces

✖ 1 problem (1 error, 0 warnings)
```

Ideally, there should be a way to except properties from the no-multi-spaces check if key-spacing has align mode enabled.

Working on this.
